shared/desktop/app/menu-bar.desktop.tsx

```diff
@@ -6,12 +6,19 @@ import {resolveImage, resolveRootAsURL} from './resolve-root.desktop'
 import {showDevTools, skipSecondaryDevtools} from '../../local-debug.desktop'
 import logger from '../../logger'
 
-const htmlFile = resolveRootAsURL('dist', `menubar${__DEV__ ? '.dev' : ''}.html`)
+const htmlFile = resolveRootAsURL('dist', `menubar${__DEV__ ? '.dev' : ''}.html?param=`)
```

Comment:
so the url scheme matches what we do elsewhere. the main thread finds things from url now